components/http_request.rst

```diff
@@ -217,6 +217,29 @@ as seen below.
         # Will send:
         # {"key": 42.0, "greeting": "Hello World"}
 
+GET values from a JSON body response
+************************************
+
+The server returns a response in a JSON object over HTTP like this:
+``{"type":"0","ch":"0","mode":"40","loop":"0","eq":"0","status":"play","curpos":"7133","offset_pts":"7133","totlen":"0","Title":"556E6B6E6F776E","Artist":"556E6B6E6F776E","Album":"556E6B6E6F776E","alarmflag":"0","plicount":"0","plicurr":"0","vol":"42","mute":"0"}``
```

Comment:
Can you please simplify this example to match those above? esphome.io domain, a very short data object, etc?

Reply:
I thought a real world working example is always better.

Reply:
While that can be true, a short, concise, easy to understand example is always better for users.

Reply:
Done.